components/RecordList.php

```diff
@@ -266,9 +272,14 @@ protected function listRecords()
 
         $model = new $modelClassName();
         $scope = $this->getScopeName($model);
+        $scopeValue = $this->property('scopeValue');
 
         if ($scope !== null) {
-            $model = $model->$scope();
+            if ($scopeValue !== null) {
```

Comment:
This is good thinking on your part, but unnecessary. PHP can send more arguments than the method asks for with no problems, but if the method asks for more arguments than are sent, that will cause a problem. You should be fine to just always do `$model->$scope($scopeValue)`